### PR TITLE
[SW-2138] Apply spotless formatting to gradle files

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: benchmarks/build.gradle [hunk 1]
```diff
@@ -42,19 +42,20 @@ task substituteTerraform(dependsOn: copyTerraform) {
...
-                .replaceAll("SUBST_PACKAGE_FILE", "${configurations.sparklingWaterAssemblyJar.singleFile}")
-                .replaceAll("SUBST_BENCHMARKS_FILE", "$buildDir/libs/sparkling-water-benchmarks_$scalaBaseVersion-${version}.jar")
-                .replaceAll("SUBST_H2O_VERSION_NAME", h2oMajorName)
-                .replaceAll("SUBST_H2O_VERSION", h2oVersion)
-                .replaceAll("SUBST_H2O_BUILD", h2oBuild)
-                .replaceAll("SUBST_SW_VERSION", version.toString())
-                .replaceAll("SUBST_SCALA_VERSION", scalaBaseVersion)
-                .replaceAll("SUBST_EMR_VERSION", supportedEmrVersion)
-            
+                    .replaceAll("SUBST_PACKAGE_FILE", "${configurations.sparklingWaterAssemblyJar.singleFile}")
```
Comment (mn-mikke) [inline benchmarks/build.gradle]: Isn't such indentation too much?

File: py/build.gradle [hunk 2]
```diff
@@ -236,102 +239,108 @@ else:
...
 if (project.property("testMojoPipeline") == "true") {
-    testPython.dependsOn testPythonMojoPipeline
+testPython.dependsOn testPythonMojoPipeline
 }
 
-task integTestPython(type: PythonTask, dependsOn: [distPython, checkSparkVersionTask]) {
-    extraArgs(*createIntegTestArgs())
-    command = "tests/test_runner.py 'tests/integration --ignore=tests/integration/external_only'"
+task integTestPython(type: PythonTask, dependsOn: [
+distPython,
```
Comment (mn-mikke) [inline py/build.gradle]: indentation here, would be nice. Also the original formatting on one line wasn't bad.

File: py/build.gradle [hunk 3]
```diff
@@ -236,102 +239,108 @@ else:
...
+checkSparkVersionTask
+]) {
+extraArgs(*createIntegTestArgs())
```
Comment (mn-mikke) [inline py/build.gradle]: indentation of task body would be also great.

File: r/build.gradle [hunk 4]
```diff
@@ -73,12 +73,20 @@ task test(type: Exec, dependsOn: distR) {
...
+        "R",
+        "-e",
+        "install.packages(\"h2o\", type = \"source\", repos = \"http://h2o-release.s3.amazonaws.com/h2o/${majorName}/${h2oBuild}/R\")"
+    ])
 }
 
 task installRSparklingPackage(type: Exec, dependsOn: distR) {
     workingDir project.buildDir
-    commandLine getOsSpecificCommandLine(["R", "-e", "install.packages(\"rsparkling_${version}.tar.gz\", type=\"source\", repos=NULL)"])
+    commandLine getOsSpecificCommandLine([
```
Comment (mn-mikke) [inline r/build.gradle]: It would be more legible if it was on one line

**Comment by jakubhava** [comment]
We can configure this formatter a lot, will play with it

File: r/build.gradle (hunk 4 above)
Comment (jakubhava) [reply r/build.gradle]: Yup, fixed

File: benchmarks/build.gradle (hunk 1 above)
Comment (jakubhava) [reply benchmarks/build.gradle]: Should be fixed

File: py/build.gradle (hunk 2 above)
Comment (jakubhava) [reply py/build.gradle]: Seems it's broken only in py/build.gradle, will investigate

**Comment by jakubhava** [comment]
@mn-mikke should be hopefully better on all fronts, please check :)

File: py/build.gradle (hunk 3 above)
Comment (jakubhava) [reply py/build.gradle]: Yup, was broken in `py`

**Pull request merged**